roles/tas_single_node/defaults/main.yml

```diff
@@ -51,6 +59,9 @@ tas_single_node_tsa_ca_passphrase: rhtas
 tas_single_node_tsa_signer_passphrase: rhtas
 tas_single_node_ct_logprefix: rhtasansible
 
+tas_single_node_fulcio_private_key: ""
+tas_single_node_fulcio_root_ca: ""
```

Comment:
We now have the nested values under `tas_single_node_fulcio`, so we can remove these ones, right?